elasticsearch_dsl/field.py

```diff
@@ -345,3 +385,12 @@ class DateRange(Field):
 
 class Join(Field):
     name = 'join'
+
+class TokenCount(Field):
+    name = 'token_count'
+
+class Murmur3:
+    name = 'murmur3'
+
+class Percolator:
+    name = 'percolator'
```

Comment:
I wonder if we should deserialize the percolator query into a `Query` object, what do you think?